src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -776,21 +780,31 @@ const parseKey = debounce(500, (key, setKeyObject, setKeyInvalid) => {
                 setKeyInvalid(true);
                 console.warn("Could not validate the public key");
             });
-});
+    };
+
+    const debouncedParseKey = useInit(() => debounce(500, parseKey));
```

Comment:
I need to wrap my brain around this, this makes it so that `debounceParseKey` lives "forever"? And thus keeps debouncing?

Reply:
Seems an alternative is to use `useMemo` or `useCallback`

https://dmitripavlutin.com/react-throttle-debounce/

Reply:
But do we need an alternative? :-) `useInit` avoids both the performance problem of `useCallback` and the threat of `useMemo` to sometimes throw away the memoized value.

Reply:
No, was just curious :)